.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What I use to debug the backend. The port number doesn't work now that `nx` assigns it randomly, but if we can make it static again, it should work just fine